gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/bin/gravitee

```diff
@@ -28,9 +28,9 @@ fi
 
 # Searching for configuration 
 GRAVITEE_OPTS=""
-if [ -f "/etc/gravitee-management/gravitee.yml" ]
+if [ -f "/opt/graviteeio-management-api/config/gravitee.yml" ]
 then
-	GRAVITEE_OPTS="-Dgravitee.conf=/etc/gravitee-management/gravitee.yml"
+	GRAVITEE_OPTS="-Dgravitee.conf=/opt/graviteeio-management-api/config/gravitee.yml"
```

Comment:
Looks like we have to do the same here also https://github.com/gravitee-io/gravitee-api-management/blob/a88b259ade0d8373f0b188f3d7cf403365cfd08d/gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/bin/gravitee#L31

Reply:
The corresponding path in the gateway pod is this one "/opt/graviteeio-gateway/config". But i didn't had any problem with this pod - maybe there the path or the GRAVITEE_OPTS is set somewhere else...

Reply:
About this, there is a default mechanism here: https://github.com/gravitee-io/gravitee-node/blob/master/gravitee-node-container/src/main/java/io/gravitee/node/container/AbstractContainer.java#L53

If the `gravitee.conf` is not set, the config file is read from `gravitee.home`.

IMHO your modification can be considered as a breaking change for some users (for instance who are running Gravitee APIM from zip files) who have a custom config file in this folder.